app/css/obsadmin.css

```diff
@@ -41,30 +42,30 @@ header .logo img {
   width: 120px;
 }
 
-.nav-header a{
+.nav-header a {
   color: white;
 }
 
-.glyphicon-user:before{
+.glyphicon-user:before {
   color: white;
 }
 
-.glyphicon-log-out:before{
-  color:white;
+.glyphicon-log-out:before {
+  color: white;
 }
 
 .fa-user:before, .fa-map-marker:before, .fa-sign-out:before {
-    color: white;
-    margin-right: 10px;
-    font-size:20px;
+  color: white;
+  margin-right: 10px;
+  font-size: 20px;
 }
 
 .fa-user:before, .fa-map-marker:before {
-    margin-right: 10px;
```

Comment:
Are all changes in this file and others simply about "Add plus icon for addresses"?

Reply:
not really, I changed the indentation and removed some unused code, just some bit of clean up.

Reply:
That should be done in a separate ticket and pull request. This one should have only the bare minimum required by the ticket.

Reply:
The only added feature is the plus icon for addresses, the rest is just code clean up.

Reply:
it's just indentation fixes not a feature

Reply:
@dkayiwa you mean  a ticket for indentation?

Reply:
yes